graphicsmagick/graphicsmagick.go

```diff
@@ -29,23 +30,44 @@ const (
 //
 // Params:
 //
-// - width / height: sizes for "-resize" argument (both optionals)
+// - ignore_ratio: "!" for "-resize" argument
 //
 // - fill: "^" for "-resize" argument
 //
-// - ignore_ratio: "!" for "-resize" argument
-//
 // - only_shrink_larger: ">" for "-resize" argument
 //
 // - only_enlarge_smaller: "<" for "-resize" argument
 //
-// - background: color for "-background" argument, 3/4/6/8 lower case hexadecimal characters
+// - extent: "-extent" param, uses width/height params
+//
+// - width / height: sizes for "-resize" argument (both optionals)
 //
-// - extent: "-extent" param, uses width/height params and add "-gravity center" argument
+// - quality: "-quality" param
+//
+// - background: color for "-background" argument, 3/4/6/8 lower case hexadecimal characters
 //
 // - format: "-format" param
 //
-// - quality: "-quality" param
+// - gravity: "-gravity" param, default is Center
+//
+// - crop: "-crop" param with "+repage"
+//
+// - rotate: "-rotate" param
+//
+// - monochrome: "-monochrome" param
+//
+// - grey: "-colorspace" param with "GRAY"
+//
+// - no_strip: "-strip" param
+//
+// - trim: "-trim" param
+//
+// - no_interlace: "-interlace" param with "Line"
+//
+// - flip: "-flip" param
+//
+// - flop: "-flop" param
+
```

Comment:
https://travis-ci.org/pierrre/imageserver/jobs/93622418#L760-L762
Remove this new line.